include/cadmium/modeling/devs/component.hpp

```diff
@@ -109,11 +111,13 @@ namespace cadmium {
 		 * @throws CadmiumModelException if there is no input port with the provided ID.
 		 */
         [[nodiscard]] std::shared_ptr<PortInterface> getInPort(const std::string& id) const {
-            try {
```

Comment:
Just curious: why did you change the getter functions this way? Is this more performant?

Reply:
This was changed in an attempt to reduce the number of required headers further. The original implementation used the 'out_of_range' exception which required the stdexcept.h header. On making this change only the exception header is needed.